extensions/wikia/AdEngine/js/provider/gpt/adSizeFilter.js

```diff
@@ -30,6 +31,33 @@ define('ext.wikia.adEngine.provider.gpt.adSizeFilter', [
 		return goodSizes;
 	}
 
+	function filterOutIncontentLeaderboardSizes(sizes) {
+		log(['filterOutIncontentLeaderboardSizes', sizes], 'debug', logGroup);
+		var goodSizes = [], i, len, currentWidth;
+
+		currentWidth = doc.documentElement.offsetWidth;
+
+		if (currentWidth > 1595) {
```

Comment:
Let's try using [wikia.breakpointsLayout](https://github.com/Wikia/app/blob/dev/resources/wikia/modules/breakpointsLayout.js) module here instead of hardcoded numbers. I'm also thinking about refactoring it all and removing duplicated code but I can handle it within [ADEN-2987](https://wikia-inc.atlassian.net/browse/ADEN-2987).